ports/glutin/headless_window.rs

```diff
@@ -196,8 +211,8 @@ impl WindowMethods for Window {
                 self.context.context,
                 buffer.as_mut_ptr() as *mut _,
                 gl::UNSIGNED_BYTE,
-                self.context.width as i32,
-                self.context.height as i32,
+                self.context.width.get() as i32,
+                self.context.height.get() as i32,
             );
             assert_ne!(ret, 0);
```

Comment:
This assertion is now firing because we're calling OSMesaMakeCurrent with a size that is different than the original size the context was created with. I think we will need to recreate the context when resizing the headless window.